### PR TITLE
DOC-5444 debug tsdump

### PR DESCRIPTION
**Pull request by andf-crl**
Addresses: DOC-5444

- Added new reference page for `cockroach debug tsdump` and added to various commands list includes.
	- Added to docs for v22.2, v22.1, and v21.2. All changes identical across versions.

Staging

[v22.2/cockroach-debug-tsdump.md](https://deploy-preview-15980--cockroachdb-docs.netlify.app/docs/stable/cockroach-debug-tsdump.html) | [v22.2/cockroach-commands.md](https://deploy-preview-15980--cockroachdb-docs.netlify.app/docs/stable/cockroach-commands.html#commands)

### CODE REVIEW

**Review by nickvigilante — APPROVED** [review]
Approved for Vale changes only.

**Comment by andf-crl** [comment]
@rupertharwood-crl mind reviewing this, since you requested it? Or feel free to pass to an engineer of your choosing. I drew content from Nigel's and Tobias' internal resources, as linked in DOC-5444. Thanks!!

**Review by rupertharwood-crl — APPROVED** [review]
This looks great. One thing that would be a big win would be to make a note of exactly what information is contained in the tsdump for customers who are concerned about PII. I believe this is entirely metadata, but does contain things like hostname and ip address for graph labeling.

**Comment by andf-crl** [comment]
@rupertharwood-crl is investigating what such a note might include, but otherwise ready for copy review. Thanks for the once-over Rupert, will add any follow-ups you have regarding your comment before merging. Thanks!

**Review by thtruo — APPROVED** [review]
LGTM

**Comment by shannonbradshaw** [comment]
@andf-crl are you waiting on @rupertharwood-crl for more information regarding the note he mentions? Is there a reason we can't merge and then add his additions as follow up?

**Comment by andf-crl** [comment]
@shannonbradshaw I'm waiting on docs review for this one. Would you like me to skip?

**Comment by shannonbradshaw** [comment]
I just reviewed. LGTM.

**Pull request merged**
